packages/nextjs/pages/index.tsx

```diff
@@ -269,4 +280,20 @@ const Home: NextPage = () => {
   );
 };
 
+export const getStaticProps: GetStaticProps<{ stats: Stats }> = async () => {
+  const res = await fetch("https://buidlguidl-v3.ew.r.appspot.com/api/stats");
```

Comment:
Sorry, I missed this in the other review.

Can we move `https://buidlguidl-v3.ew.r.appspot.com/` to an envvar? Like BG_BACKEND or BG_BACKEND_API or whatever.

What do you think?

Reply:
Yup !! used `BG_BACKEND_API=https://buidlguidl-v3.ew.r.appspot.com ` 🙌